src/app/app.component.html

```diff
@@ -1,42 +1,43 @@
-<section class="black">
-  <div class="container width">
-    <div class="fourteen unit column">
-      <div class="one half unit row">
-        <div class="one half unit column">
-          <img draggable="false" src="/assets/images/logo.svg" alt="Hack Cambridge Logo">
+<header class="black">
+    <div class="container width">
+        <div class="three unit row">
+            <div class="one half unit column">
+                <img draggable="false" src="/assets/images/logo.svg" alt="Hack Cambridge Logo">
+            </div>
+            <div class="six unit column">
+                <h2>Hack Cambridge<br><span class="pixel">Ternary</span></h2>
+            </div>
         </div>
-        <div class="six unit column">
-          <h2>Hack Cambridge<br><span class="pixel">Ternary</span></h2>
+        <div class="one unit row">
+            <div class="one half unit column"></div>
+            <div class="six unit column">
+                <h3>{{ companyName ? "Hi " + companyName + "!" : 'Sponsorship Portal' }}</h3>
+            </div>
         </div>
-      </div>
     </div>
+</header>
 
-    <div class="one unit row">
-      <div class="one half unit column"></div>
-      <div class="six unit column">
-        <h3>{{ companyName ? companyName : 'Sponsorship Portal' }}</h3>
-      </div>
+<router-outlet></router-outlet>
+<section class="black">
+    <div class="container width">
+        <div class="one half unit row">
+            <div class="unit column grows">
+                <a rel="external" href="https://hackcambridge.com/">Hack Cambridge Main Site</a>
+            </div>
+        </div>
```

Comment:
Okay, then we need something else to go in this section (because of the unpredictable structure just before the footer which prevents the footer from knowing if a coloured section is above it). Let's discuss.